src/main/scala/org/bblfsh/client/v2/BblfshClient.scala

```diff
@@ -0,0 +1,161 @@
+package org.bblfsh.client.v2
+
+import java.nio.ByteBuffer
+
+import com.google.protobuf.ByteString
+import com.google.protobuf.duration.Duration
+import io.grpc.ManagedChannelBuilder
+import gopkg.in.bblfsh.sdk.v2.protocol.driver.{ParseRequest, ParseResponse, DriverGrpc, SupportedLanguagesRequest, SupportedLanguagesResponse, VersionRequest, VersionResponse, DriverHostGrpc}
+
+import org.bblfsh.client.v2.libuast.Libuast
+
+
+class BblfshClient(host: String, port: Int, maxMsgSize: Int) {
+  private val DEFAULT_TIMEOUT_SEC = 5
```

Comment:
Do we have to have a default? And if so, can we make it fairly long? As we've seen, it's hard to debug weird timeout failures deep inside the plumbing. Assuming this is only here as a safety backstop so a bug doesn't wedge us forever, it should be fine to have it be like 1m or 2m. "This always fails after around 1 minute" is easy to diagnose vs. "This often but not always times out around 5 seconds".

Reply:
I know that Github does not make it simpler to distinguish, but this is not a change but rather the same code as before. There is a TODO for dealing with timeouts, both in code and an the umbrella issue description, so I've attached the link in there of feedback not to get lost.

I appreciate the feedback, but if possible, would prefer focusing this and few subsequent PRs on functionality related to SDK/libuast move.

Reply:
> [I]f possible, would prefer focusing this and few subsequent PRs on functionality related to SDK/libuast move.

Yes, that makes sense to me. And as you observed, I simply did not notice that this was the same as the previous behaviour.